concourse/tasks/daily-statistics.yml

```diff
@@ -25,4 +25,4 @@ run:
       cf auth
       cf t -o "$CF_ORG" -s "$CF_SPACE"
 
-      cf ssh $CF_APP_NAME -c "/tmp/lifecycle/launcher /home/vcap/app 'rake statistics:general' ''" > statistics/output.json
+      cf ssh $CF_APP_NAME -c "/tmp/lifecycle/launcher /home/vcap/app 'rake statistics:general' ''" > statistics/output.json 2>/dev/null
```

Comment:
This won't affect the validity of the json file, because `> statistics.output.json` doesn't capture stderr.

Reply:
Ah, yes, of course.  I'll drop this commit.